instrumentation/rocketmq-client-4.8/javaagent/rocketmq-client-4.8-javaagent.gradle

```diff
@@ -17,5 +17,6 @@ dependencies {
 }
 
 tasks.withType(Test) {
+  jvmArgs += ["--add-exports", "java.base/jdk.internal.ref=ALL-UNNAMED"]
```

Comment:
Thanks @addname - is this needed for rocketmq server, not client? If so, is it possible to use test containers with docker instead of having to use this custom Java config? We use it in a few tests.

Reply:
`brokerController.shutdown()` needs to be set `--add-exports`

Reply:
Sounds like we'd be able to use testcontainers then - we can do it in another PR though